src/app/services/mini-games-http.service.ts

```diff
@@ -9,12 +11,11 @@ import { ApiService } from '../core/services/api.service';
 export class MiniGamesHttpService {
   constructor(private http: HttpClient, private api: ApiService) {}
 
-  getWords() {
-    return this.http.get(`${this.api.baseApiUrl}/words`);
+  getWords(group: number = 0, page: number = 0): Observable<IWord[]> {
+    return this.http.get<IWord[]>(`${this.api.baseApiUrl}/words?group=${group}&page=${page}`);
```

Comment:
You can use ```params``` property
```
const params = { group, page };
this.http.get<IWord[]>(`${this.api.baseApiUrl}/words`, { params });

Reply:
These changes lead to the error: 
Type 'Observable<ArrayBuffer>' is not assignable to type 'Observable<IWord[]>'.
  Type 'ArrayBuffer' is missing the following properties from type 'IWord[]': length, pop, push, concat, and 25 more.
 
I will leave it as it is and use the newly created words api service when it is merged.